mod/NomaiTextQoL.cs

```diff
@@ -1,42 +1,146 @@
-﻿using HarmonyLib;
+using HarmonyLib;
+using System.Linq;
+using System;
+using UnityEngine;
+using System.Collections.Generic;
 
 namespace ArchipelagoRandomizer
 {
     [HarmonyPatch]
     internal class NomaiTextQoL
     {
         public static bool AutoNomaiText;
+        public static bool ColorNomaiText = true;
         public static float TranslateTime = 0.2f;
 
+        public static Dictionary<string, Location> ManualScrollLocations = new()
+        {
+            { "Arc_BH_City_School_BigBangLesson", Location.BH_SOLANUM_REPORT },
+            { "Arc_TT_Tower_CT", Location.AT_HGT_TOWERS },
+            { "Arc_TT_Tower_BH_1", Location.AT_BH_TOWER }
+        };
```

Comment:
We should move this dict to LocationTriggers so it can be shared by the code that checks these locations.

Reply:
Sure, done